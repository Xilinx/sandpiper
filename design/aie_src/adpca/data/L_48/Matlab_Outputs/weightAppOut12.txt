-9.342029572e+00 1.220045090e+00
-2.569981575e+00 -5.167224884e+00
-5.499334335e+00 2.395443916e-01
