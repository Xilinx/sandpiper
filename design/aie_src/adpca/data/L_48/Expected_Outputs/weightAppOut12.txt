-9.341737747e+00 1.219889641e+00 
-2.569715977e+00 -5.167057037e+00 
-5.499311924e+00 2.395451069e-01 
